CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 - **Breaking change** `api.derive.contract` is now `api.derive.contracts` to align with the substrate 2.x rename. (Feture detection is used so it supports both 1.x and 2.x chains)
 - **Breaking change** The api now uses the module name instead of the prefix to generate the storage methods. The methods of the grandpa module changed from `api.query.grandpaFinality` to `api.query.grandpa`. 
 - Update with latest substrate 2.x types
-- Support latest substrate 2.x v6 metadata with module constants
+- **Breaking change** Support latest substrate 2.x v6 metadata with module constants. Some rarely changing and frequently queried parameters have been moved from storage to constants. e.g. `launchPeriod` is directly queriable with `api.constants.launchPeriod`, no longer `api.query.democracy.launchPeriod`.
```

Comment:
Ah, sorry, in my other post I meant the change `StorageFunction -> StorageEntry`. This part should be documented too , but I guess in a later PR, when https://github.com/polkadot-js/api/issues/1043 is definitely closed

Reply:
ah my bad